ArduCopter/wscript

```diff
@@ -45,5 +45,5 @@ def build(bld):
         program_name='arducopter-heli',
         program_groups=['bin', 'heli'],
         use=vehicle + '_libs',
-        defines=['FRAME_CONFIG=HELI_FRAME'],
+        defines=['FRAME_CONFIG=HELI_FRAME', 'HELI_BUILD'],
```

Comment:
I must say that I'm not a fan of having two definitions which have the same function.  We should at least raise an issue to get rid of one of them "FRAME_CONFIG?" in a follow-up PR

Reply:
Issue submitted.